judo-ui-react/src/main/resources/actor/src/components/dialog/FilterDialog.tsx.hbs

```diff
@@ -195,7 +198,7 @@ const FilterInput = ({ filter, setFilterValue, valueId }: FilterInputProps) => {
           //   return (
           //     <TextField
           //       className={valueId}
-          //       label={filter.filterOption.attributeName}
+          //       label={filter.filterOption.label ?? filter.filterOption.attributeName}
```

Comment:
Why is this part commented? @noherczeg

Reply:
Time was never supported for filtering lol. Will be introduced the first time it's needed.